e2e/testdata.go

```diff
@@ -96,3 +121,133 @@ func getSamplePvc(size, name string, volumemode k8sv1.PersistentVolumeMode, stor
 	}
 	return pvc
 }
+
+// ExecOptions passed to ExecWithOptions
+type ExecOptions struct {
+	Command       []string
```

Comment:
The exec commands defined here are already available in k8s.io/kubernetes/test/e2e/framework. Please use the functions from that package instead.